CONTRIBUTING.md

```diff
@@ -27,7 +27,7 @@ This will be necessary for the pull request in the
 
 ## Second step: Add your Project details
 
-Simply copy the file `src/TEMPLATE.yaml` into the `project` folder
+Simply copy the file [`TEMPLATE.yaml`](./TEMPLATE.yaml) into the `project` folder
```

Comment:
@AblingerOscar sorry, there's an inconsistency here. Either we `git mv src project`, or let's rewrite this line to "into the \`src/\` folder"